libc-test/build.rs

```diff
@@ -2293,6 +2294,10 @@ fn test_linux(target: &str) {
             t if mips32_musl && t.starts_with("IFF") => true,
             "MFD_HUGETLB" | "AF_XDP" | "PF_XDP" if mips32_musl => true,
 
+            // FIXME: added in Linux 5.1 but not yet in glibc or musl. Enable in linux_fcntl tests
+            // once available.
+            "F_SEAL_FUTURE_WRITE" => true,
```

Comment:
Search for "// Require Linux kernel 5.x:" above. Could you add this there, e.g. adding a new "// Require Linux kernel 5.1" followed by this constant ?

Reply:
done